common/src/main/java/org/jboss/pnc/common/scm/GerritScmUrlGenerator.java

```diff
@@ -74,42 +58,40 @@ public String generateGerritGitwebLogUrl(@NotNull String gerritUrl, String ref)
     }
 
     /**
-     * Generate an https URL generated from the 'git clone' Gerrit url. The url generated points to the specific commit
+     * Generate a https URL generated from the 'git clone' Gitweb url. The url generated points to the specific commit
```

Comment:
it's an "aych tee tee pee es"
```suggestion
     * Generate an https URL generated from the 'git clone' Gitweb url. The url generated points to the specific commit
```

Reply:
damn IntelliJ was telling me it was wrong